modules/quadbin/snowflake/sql/QUADBIN_BBOX.sql

```diff
@@ -0,0 +1,148 @@
+-- This method should call QUADBIN_TOZXY, but that causes SnowFlake to crash,
+-- so we are computing the ZXY here, in-place
+CREATE OR REPLACE FUNCTION QUADBIN_BBOX
+(quadbin INT)
+RETURNS ARRAY
+AS $$
+WITH __interleaved AS (
```

Comment:
Can you add a special case to return NULL when quadbin is NULL? We're doing that in the other providers. (I think I was the one that dropped the check when I looked for a workaround for the SF crash)